scripts/set_s3_creds

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python
```

Comment:
Looks good, but probably this script should go into gbdxcli:

https://github.com/digitalglobe/gbdxcli

It doesn't quite fit into the base functionality repo.